modules/publication/ajax/FileUpload.php

```diff
@@ -456,8 +470,10 @@ function notifyEdit($pubID)
     $emailData = array();
 
     $data = $db->pselectRow(
-        "SELECT Title, DateProposed, LeadInvestigatorEmail ".
-        "FROM publication ".
+        "SELECT Title, DateProposed, pc.Email as LeadInvestigatorEmail " .
+        "FROM publication p " .
+        "LEFT JOIN publication_collaborator pc ".
+        "ON p.LeadInvestigatorID=pc.PublicationCollaboratorID " .
         "WHERE PublicationID=:pubID",
         array('pubID' => $pubID)
     );
```

Comment:
Lots of common code in various notifyX() function. Could use factoring for subparts or an extra parameter to a single function